querybook/webapp/components/DataDocSchedule/DataDocScheduleForm.tsx

```diff
@@ -284,6 +270,148 @@ export const DataDocScheduleForm: React.FunctionComponent<
     );
 };
 
+function getHelpInfo(selectedWith) {
+    const helpers = {
+        slack: `You can type slack channels here and you can use comma to separate multiple channels`,
```

Comment:
slack username or slack channel names

also how do we differentiate a username and a slack channel name?  probably we can ask user to type prefix `@` and `#` for them separately